.github/workflows/Basic.yml

```
@@ -18,15 +18,15 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.87.0
+          toolchain: 1.88.0
           target: wasm32-unknown-unknown
           override: true
 
       - name: Run all Rust tests
         uses: actions-rs/cargo@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)